print-apps/oereb/tocReduced.jrxml

```diff
@@ -14,7 +14,6 @@
 	<property name="com.jaspersoft.studio.unit.columnSpacing" value="pixel"/>
 	<style name="Default" isDefault="true" fontName="Cadastra" fontSize="8.5"/>
 	<parameter name="SUBREPORT_DIR" class="java.lang.String"/>
-	<parameter name="isReduced" class="java.lang.Boolean"/>
```

Comment:
Funny to find this here as a parameter. It is never used.
Now it is gone that is good :+1: